### PR TITLE
feat: add tenv

### PR DESCRIPTION
**Pull request by kvendingoldo**

### CODE REVIEW

**Comment by wmariuss** [comment]
I think is better to have a section like `Productivity Tools` with the list of tools which increase productivity like the tool you mentioning.

**Comment by kvendingoldo** [comment]
done

File: README.md [hunk 1]
```diff
@@ -171,6 +171,10 @@ DevOps is the combination of cultural philosophies, practices, and tools that in
 - [Meshery](https://meshery.io/) - An open source, cloud native manager that enables the design and management of all Kubernetes-based infrastructure and applications.
 - [Digger](https://digger.dev) - Open Source Infrastructure as Code management tool that runs within your CI/CD system.
 
+## Productivity Tools
```
Comment (wmariuss) [inline README.md:174]: Cn you also add the section on the `CONTENTS`? Ex. here https://github.com/wmariuss/awesome-devops/blob/main/README.md#contents

Comment (wmariuss) [reply README.md:174]: I will add it.

**Review by wmariuss — APPROVED** [review]

**Pull request merged**